files/root/index.html

```diff
@@ -111,6 +111,7 @@
         cursor: pointer;
       }
     </style>
+    <link rel="icon" type="image/svg+xml" href="/lnme/favicon.svg" />
```

Comment:
👍 we should use that also as `og:image` further up in line 9

Reply:
Updated, removed the zap.svg and rebuild the generated code with rice

Reply:
should we use the full lnme-icon as `og:image` ? 

in Alby for example: 
<img width="404" alt="image" src="https://user-images.githubusercontent.com/318/144595208-82227673-4919-40e4-9fa1-3b77ce0e93f3.png">
vs.
<img width="400" alt="image" src="https://user-images.githubusercontent.com/318/144595239-02cdabec-1445-4d20-9b39-31527eb7752e.png">